src/main/scripts/cd/release-generate-docs.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+# This script generates Release api documentation and saves it to develop branch
+
+cd $TRAVIS_BUILD_DIR && git checkout master
+
+SCALECUBE_CFG_SERVICE='/tmp/scalecube-cfg-service'
+SCALECUBE_CFG_SERVICE_DOCS='/tmp/scalecube-repo/configuration-service/'
+RELEASE=$(mvn -q -Dexec.executable="echo" -Dexec.args='${project.version}' --non-recursive exec:exec)
+RELEASE_TEMPLATE='0.0.0-CURRENT'
+
+# clone needed repositories
+git clone git@github.com:scalecube/scalecube.github.io.git /tmp/scalecube-repo
+git clone git@github.com:scalecube/scalecube-configuration-service.git $SCALECUBE_CFG_SERVICE
```

Comment:
@skolomiiets is it needed? we already are cloning this (in travis)

Reply:
We need 2 branches at the same time: release branch for generating documentation and develop branch for adding this documention